orchagent/copporch.cpp

```diff
@@ -75,12 +77,68 @@ std::map<string, sai_packet_action_t> packet_action_map = {
     {"transit", SAI_PACKET_ACTION_TRANSIT}
 };
 
+std::vector<sai_hostif_trap_id_t> default_trap_ids = {
+    SAI_HOSTIF_TRAP_ID_TTL_ERROR
+};
+
+const string default_trap_group = "default";
```

Comment:
This is setting the trap channel for the trap_ids, by default they all go to callback channel. What we really want here is to set NETDEV as the default channel for all trap_ids. There is no good way to do it now except enumerating all of them, but not all vendor SAI supports all those traps, so it will cause issue if we use that way. Since we mostly care about TTL_ERROR, we just set it here. Other traps will be explicit set by the configuration file.